.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+ignore = E203,E501,W605,F541
+max_line_length = 160
```

Comment:
We might want to unify the line lengths, currently using `max_line_length=160` for `flake8` but 100 for `black`.

Reply:
Ok, will unify to length=100 for both